src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -157,7 +159,7 @@ private void setWikiTitle(Article article) {
 	 */
 	private void setIsList(Article article) {
 		for (String list : locale.getListIdentifiers()) {
-			if (StringUtils.startsWithIgnoreCase(article.getTitle(), list)) {
+			if (StringUtils.startsWithIgnoreCase(article.getTitle(), list + " ")) {
```

Comment:
It might be good to set the white space either as a constant or a word delimiter (I'd imagine some other languages might have different separators )?